packages/amplify-category-api/provider-utils/awscloudformation/service-walkthroughs/apigw-walkthrough.js

```diff
@@ -1,6 +1,7 @@
 const inquirer = require('inquirer');
 const pathLib = require('path');
 const fs = require('fs-extra');
+const { ServiceName: FunctionServiceName } = require('amplify-category-function');
```

Comment:
within the function category it makes sense to just be called the ServiceName because it's obviously dealing with functions. But since it's being imported in a different package here, I think it makes sense to alias it to FunctionServiceName to make it clear which category it's dealing with (so you don't have to scroll to the top to see where it's imported from)